<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>731</width>
    <height>491</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>731</width>
    <height>491</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url('data/textures/background.jpg');
background-repeat: no-repeat; 
background-position: center;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friends_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Друзья</string>
    </property>
   </widget>
   <widget class="QPushButton" name="questionnaires_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Анкеты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="groups_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Группы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Вход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="account_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Аккаунт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="throw_dice_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* border-color: rgb(142, 74, 25);
border: 2px solid rgb(142, 74, 25); */
/* background-color: rgb(142, 74, 25); */

QPushButton {
	color: rgb(240, 155, 89);
    background-color: transparent; /* Делаем фон прозрачным */
    border: 2px solid rgb(240, 155, 89);  /* Цвет границы */
    border-radius: 15px; /* Закругление углов (по желанию) */
    padding: 5px; /* Отступы (по желанию) */
	/* border-image: linear-gradient(to right, red, yellow) 5; */
}

QPushButton:hover {
    background-color: rgba(54, 21, 8, 0.5); 
} 
</string>
    </property>
    <property name="text">
     <string>Кинуть дайс</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
